### DISCUSSION TITLE
exporter/jaeger - Inside otel collector config for exporter section of jaeger is there a way to hit port 6832/6831 (udp ports).

### DISCUSSION THREAD

**Discussion by SunilManthenaG01** [open-telemetry/opentelemetry-collector#3080] [Q&A]
Currently in all examples and demo files of the otel-collector-config file, inside the exporter jaeger section, we are hitting HTTP port 14250. Is there a way to hit the udp port 6832/6831 ?

[the remark this answers is not in the document]
**Reply by jpkrohling** [reply]
No, Jaeger exporters in the OpenTelemetry Collector are only able to send data via gRPC (14250) to a remote Jaeger Collector. The idea behind sending spans via UDP is for *clients* to send data to agents located very close to the workload, in order to offload the business application from concerns such as buffering/batching, retry, security, ...

That said, why would you want to use UDP between an OpenTelemetry Collector and a Jaeger Collector?
